xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-macro/xwiki-platform-notifications-macro-ui/src/main/resources/XWiki/Notifications/Code/Macro/NotificationsMacro.xml

```diff
@@ -422,8 +422,11 @@ require(['jquery', 'XWikiAsyncNotificationsMacro', 'xwiki-events-bridge'], funct
       if (!entry.read) {
         // Style the read button
         readButton.addClass('notification-event-read-button').addClass('btn btn-xs');
-        // Insert the cross icon
-        readButton.html('$services.icon.renderHTML("check")');
+        // Insert the cross icon and an accessible label
+        var readButtonIcon = $('$services.icon.renderHTML("check")');
+        var readButtonLabel= $('&lt;span class="sr-only"/&gt;');
```

Comment:
what is `sr-only`?

Reply:
`sr-only` is a Bootstrap class standing for "screen reader only". Probably using `aria-label` instead could be considered simpler and possibly a better practice (not sure, currently investigating this), however at the moment we have no control on the attributes set by the Icon Service (I plan to create an issue about that).